Dockerfile

```diff
@@ -63,6 +63,8 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
     helm repo add aou-sas-chart https://storage.googleapis.com/terra-app-helm/aou-sas-chart && \
     helm repo update
 
+COPY ./wds-4.46.0.tgz /leonardo
```

Comment:
Seems maybe unintentional to have another hardcoded version string in the `Dockerfile`...just checking if this is what you want, or is this just a shortcut leftover from testing something?

Reply:
agree, this would hardcode Leo to a specific wds version

Reply:
sorry this is just for testing in my bee, I will go ahead and remove this (forgot this step once I completed testing)